pkg/controller/endpointslicemirroring/reconciler_test.go

```diff
@@ -43,6 +43,7 @@ func TestReconcile(t *testing.T) {
 	testCases := []struct {
 		testName                 string
 		subsets                  []corev1.EndpointSubset
+		epLabels                 map[string]string
```

Comment:
this was the simpler way I've found to add the new testcase with labels, the alternative is to replace 
`subsets                  []corev1.EndpointSubset` by `endpoints: *corev1,Endpoints` 
but that is a big change in terms of lines of code and I don't know if is worth the risk/effort :sweat_smile: